docs/quick-starts/framework/flutter/README.mdx

```diff
@@ -51,6 +53,71 @@ This tutorial will show you how to integrate Logto into your Flutter application
 
 <Organization />
 
+## Migration guide \{#migration-guide}
+
+If you are migration from a previous version of Logto Dart SDK, version < 3.0.0:
+
+1. Update your `pubspec.yaml` file to use the latest version of the Logto Dart SDK.
+
+   ```yaml title="pubspec.yaml"
+   dependencies:
+     logto_dart_sdk: ^3.0.0
+   ```
+
+2. Update the Android manifest file, replace the legacy `flutter_web_auth` callback activity with the new `flutter_web_auth_2`.
+
+   - `FlutterWebAuth` -> `FlutterWebAuth2`
+   - `flutter_web_auth` -> `flutter_web_auth_2`
+
+3. Pass the `redirectUri` to the `signOut` method.
+
+   `redirectUri` is now required when calling the `signOut` method. For iOS platform, this parameter is useless, but for Android and Web platforms which require an additional `end_session` request to clean up the sign-in session, this parameter will be used as the `post_logout_redirect_uri` parameter in the `end_session` request.
+
+   ```dart
+   await logtoClient.signOut(redirectUri);
+   ```
+
+## Troubleshooting \{#troubleshooting}
+
+### Troubleshooting Android
```

Comment:
Also no heading ID for this one? Did you add those IDs manually or use the docusaurus command?